### ISSUE TITLE
Project Tabs for Bambu Studio

### ISSUE THREAD

**Issue by philsward** [bambulab/BambuStudio#3377]
**Is your feature request related to a problem? Please describe.**
Bambu Studio will only let you work on one project at a time. You can open another instance of Bambu Studio and get the "network plugin could not install" error or just switch in and out of projects.

**Describe the solution you'd like**
Right now I'm in the middle of printing a project. I want to work on slicing, tweaking and testing a new project. Instead of opening a new instance, a new project in a new tab would be great, allowing multiple projects to be open at the same time.

**Describe alternatives you've considered**
The only workaround is a new instance of Bambu Studio.

**Additional context**
Mockup:
![bambu-studio-tabs-mockup](https://github.com/bambulab/BambuStudio/assets/3436110/1395ccac-18a7-4881-915b-c6f2172170f5)

**Comment by LJKraus** [comment]
Once a project is sent to the printer, you are free to do whatever you want in the Prepare/Preview tabs. You can start a new project, or open another project.  Once the sliced file is sent to the printer, nothing you do in Prepare/Preview will change the  print in progress.

**Comment by hexxxx** [comment]
This would be an awesome addition!

**Comment by SaltWei** [comment]
This is good suggestion. But not easy to be implemented at once.
Add into feature pool.

**Comment by JonLaliberte** [comment]
Yeah, this would be great to have. Instead I end up with this mess 😬

<img width="1114" alt="Image" src="https://github.com/user-attachments/assets/091f570f-3500-4242-9fe4-0db625f2b40b" />
